google/cloud/spanner_v1/_opentelemetry_tracing.py

```diff
@@ -15,41 +15,89 @@
 """Manages OpenTelemetry trace creation and handling"""
 
 from contextlib import contextmanager
+import os
 
 from google.api_core.exceptions import GoogleAPICallError
 from google.cloud.spanner_v1 import SpannerClient
+from google.cloud.spanner_v1 import gapic_version as TRACER_VERSION
 
 try:
     from opentelemetry import trace
     from opentelemetry.trace.status import Status, StatusCode
+    from opentelemetry.semconv.trace import SpanAttributes
 
     HAS_OPENTELEMETRY_INSTALLED = True
+    DB_SYSTEM = SpanAttributes.DB_SYSTEM
```

Comment:
1. We are changing attribute conventions here. As discussed earlier do we need to make this change under a seperate option to avoid breaking changes?
2. I also see that we might need to change span names to be consistent with this [document](https://docs.google.com/document/d/1C-2WbnKfAtl-GgOebVNYv0RdYFkb_UfyzagwJOQgxAs/edit?tab=t.0#bookmark=id.ctslkgwt101t), which is anyhow a breaking change.

So should we make these changes directly or instead protect them under an option?
My opinion is since we are refactoring observability portion, lets make all changes directly and warn the customers in release notes. Having a seperate option will make code messier.
@surbhigarg92 Can you give your thoughts on this?

Reply:
@odeke-em 
1. For attribute conventions lets enable it under a seperate flag to avoid breaking changes.
2. Lets not change any existing span names as they could bring in a breaking change. Instead can you help add additional spans that could help in debugging.